pkg/upgrade/upgrade.go

```diff
@@ -215,12 +215,16 @@ func (u *Upgrade) Run(ctx context.Context) error {
 					return err
 				}
 			}
-			// RBAC is added in 1.2.x, so if we're upgrading to that version, we need to ensure
-			// that the RBAC configmap is present.
 			if common.CheckConstraint(upgradeEverestTo, "~> 1.2.0") {
```

Comment:
What if we eventually release a `1.2.1`? Can we make this check a `1.2.x`?

Reply:
Yes, that constraint does indeed check for `1.2.x`. The `~>` is called the [pessimistic constraint operator](https://developer.hashicorp.com/terraform/language/expressions/version-constraints#version-constraint-syntax)

Reply:
Ohhh yeah, sorry for that I didn't properly check it :see_no_evil: